app/controllers/groups_controller.rb

```diff
@@ -1,10 +1,6 @@
 class GroupsController < BaseController
   layout 'darkswarm'
 
-  def index
-    @groups = EnterpriseGroup.on_front_page.by_position
-  end
-
```

Comment:
what do we render now when accessing /groups then?

Reply:
the same, this is done in the view now with inject_groups